src/Neo/ProtocolSettings.cs

```diff
@@ -157,17 +159,39 @@ public static ProtocolSettings Load(IConfigurationSection section)
                 MaxTraceableBlocks = section.GetValue("MaxTraceableBlocks", Default.MaxTraceableBlocks),
                 InitialGasDistribution = section.GetValue("InitialGasDistribution", Default.InitialGasDistribution),
                 Hardforks = section.GetSection("Hardforks").Exists()
-                    ? section.GetSection("Hardforks").GetChildren().ToImmutableDictionary(p => Enum.Parse<Hardfork>(p.Key), p => uint.Parse(p.Value))
+                    ? EnsureOmmitedHardforks(section.GetSection("Hardforks").GetChildren().ToDictionary(p => Enum.Parse<Hardfork>(p.Key, true), p => uint.Parse(p.Value))).ToImmutableDictionary()
                     : Default.Hardforks
             };
         }
 
+        /// <summary>
+        /// Explicitly set the height of all old omitted hardforks to 0 for proper IsHardforkEnabled behaviour.
```

Comment:
For compatibility reasons, explicitly set the height of  HF_Aspidochelone and HF_Basilisk hardforks to 0 for proper IsHardforkEnabled behaviour.

Reply:
Maybe directly explain that, because this will be just for these two, right?